src/share/poudriere/bulk.sh

```diff
@@ -219,6 +219,8 @@ if [ -d ${LOGD} -a ${CLEAN} -eq 1 ]; then
 	[ ${DRY_RUN} -eq 0 ] && rm -Rf ${LOGD} 2>/dev/null
 fi
 
+ingest_base_pkgs
+
 prepare_ports
```

Comment:
`ingest_base_pkgs` is called before `prepare_ports` in both commands so it should be in there. You can use `if was_a_bulk_run` if needed to limit it to package builds.